.github/workflows/release-please.yml

```diff
@@ -26,3 +31,124 @@ jobs:
             VERSION
             barretenberg.nix
             barretenberg-wasm.nix
+            
+
+  build-linux:
+    runs-on: self-hosted
+    needs: [release-please]
+    if: ${{ needs.release-please.outputs.tag-name }}
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v2
+
+    - name: Install bleeding edge cmake
+      run: |
+        sudo apt -y remove --purge cmake
+        sudo snap install cmake --classic
+        
+    - name: Create Build Environment
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install clang ninja-build yarn
+    - name: Install yarn
+      run: |
+        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+        echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+        sudo apt -y update && sudo apt -y install yarn
+    - name: Run Bootstrap
+      run: |
+        cd cpp
+        
+        cmake --preset default -DCMAKE_BUILD_TYPE=RelWithAssert
+        cmake --build --preset default --target bb
+
+        ./scripts/install-wasi-sdk.sh
+
+        cmake --preset wasm
+        cmake --build --preset wasm
+    
+    - name: Copy Barretenberg.wasm to root
+      run: cp cpp/build-wasm/bin/barretenberg.wasm ./
+      
+    - name: Compile Typescript
+      run: |
+        cd ts
+        yarn install && yarn && yarn build
+        
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '18'
+        registry-url: 'https://registry.npmjs.org'
+        
+    - name: Deploy Typescript to NPM
+      run: |
+        cd ts
+        yarn deploy
+      env:
+        NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+        
+    - name: Tar and GZip bb Binary (Ubuntu)
+      run: tar -cvzf bb-ubuntu.tar.gz cpp/build/bin/bb
+      
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      with:
+          name: release-linux-wasm
+          path: |
+            barretenberg.wasm
+            bb-ubuntu.tar.gz
+          
+  build-mac:
+    name: Build on Mac
```

Comment:
Blocking: Mac builds are currently not compiling

Reply:
This is now fixed by using macos-13 ventura and upgrading clang to 14.0.3 (xcode 14.3.1)